### PR TITLE
Dm 2993

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2993

## Description - what does this code do?
Adds max character limit to page-builder description.

## Testing done - how did you test it/steps on how can another person can test it 
1. log in as an admin and visit `/admin/pages`.
2. Either create a new page or go to the edit view of an existing page.
3. Make sure the description character counter text is working as intended (both on initial page load and as you type/paste text).
4. Add over 140 characters to the description input and try to create/update the page. Make sure you see the validation error and ensure the page was not saved.
5. Play around with creating and updating to make sure everything else still functions as intended.

## Acceptance criteria
- Add cut off word count to header text (hard cut off) 140 characters

## Definition of done
- [X] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bradjohnson92008 — APPROVED** [review]
Worked as intended - Great job Josh.

File: app/assets/javascripts/active_admin.js [hunk 1]
```diff
@@ -71,10 +76,56 @@
...
+
+  function addPageDescriptionCharacterCounterText() {
+      let hintEle = $('#page_description').next();
+      $(hintEle).append(' <span class="page-description-character-counter">(0/140 characters)</span>');
+  }
+
+  function getPageDescriptionCharacterCountOnPageLoad() {
+      let descriptionInput = $('#page_description');
+
+      let characterCounterEle = $('.page-description-character-counter');
```
Comment (emyl3) [inline app/assets/javascripts/active_admin.js:92]: on page load if a description that was previously changed before the changes were introduced had over 140 characters then the text counter isn't red but the valid color
Screen Shot 2021-11-30 at 1 52 06 PM

**Review by emyl3 — CHANGES_REQUESTED** [review]
Just found one minor thing!

File: app/assets/javascripts/active_admin.js (hunk 1 above)
Comment (JoshingYou1) [reply app/assets/javascripts/active_admin.js:92]: Nice catch Elisa. This issue has been resolved!

**Review by emyl3 — APPROVED** [review]
Looks good to me!! Great work!

**Pull request merged**